### DISCUSSION TITLE
Unistallation / nginxproxy manager

### DISCUSSION THREAD

**Discussion by fsennj** [caprover/caprover#1457] [Q&A]
Hello All,

is it possible to uninstall caprover (and all related containers like certbot, nginx.. etc) without deleting the rest of the containers?

Or is it possible to get rid of nginx / letsencrypt and use nginx proxy manager isntead?

**Comment by githubsaturn** [comment]
You can scale any container to zero, example:

docker service scale captain-captain=0
